package/MDAnalysis/analysis/base.py

```diff
@@ -219,6 +221,36 @@ def __getstate__(self):
     def __setstate__(self, state):
         self.data = state
 
+from typing import Callable, Iterable
+
+
+def delayed(obj):
```

Comment:
Why do we need our own `delayed()`.

Can't we do
```python
try:
   from dask import delayed
except ImportError:
   def delayed(*args, **kwargs):
      raise ImportError("Please install dask for this functionality")
```

Reply:
it's now used as a helper function for multiprocessing